types/marv/marv-tests.ts

```diff
@@ -0,0 +1,40 @@
+import marvDefault, { Driver } from 'marv';
```

Comment:
IMO it's not a default, it's module re-exports. (that's trivia, but could mislead readers).
it should be possible to get those I think:
```ts
migrate: [Function: migrate],
scan: [Function: scanDirectory],
drop: [Function: drop],
parseDirectives: [Function: parseDirectives]
```

Reply:
yeah, see dt-bot comment on missing symbols from 'index' scan